schema/model/identity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://schema.dash.org/dpp-0-4-0/model/identity",
+  "properties": {
+    "id": {
+      "type": "string",
+      "minLength": 42,
+      "maxLength": 44,
+      "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$"
+    },
+    "identityType": {
+      "type": "number",
+      "multipleOf": 1.0,
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "publicKeys": {
+      "type": "array",
+      "items": {
+        "$ref": "https://schema.dash.org/dpp-0-4-0/base/public-key"
+      },
+      "minItems": 1,
+      "maxItems": 100
```

Comment:
Is this supposed to be 10 or 100? [identity-create.json](https://github.com/dashevo/js-dpp/pull/77/files#diff-3542b91b9a79311ee219fa1351b7d5f4) appears to limit the publicKeys array to size 10, but I may be missing something...

Reply:
It's how many keys you generally allowed to store in your identity, while 10 is how many you can add at once. I don't know if that maks sense, those are just arbitrary numbers